docker-compose.yml

```diff
@@ -32,6 +32,9 @@ services:
       retries: 3
     environment:
       RABBITMQ_URL: ${RABBITMQ_URL}
+      DB_HOST: ${DB_HOST}
```

Comment:
Can we start to use `AUDIT_` as a prefix to these variables so they are independent as they belong to each microservice?

Reply:
added #402